README.md

```diff
@@ -1,5 +1,14 @@
 [![Build Status](https://api.travis-ci.org/abstractj/cryptoparty.png)](https://api.travis-ci.org/abstractj/cryptoparty)
 
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0  |
+| Build:          | Maven  |
+| Documentation:  | https://aerogear.org/docs/  |
```

Comment:
@danielpassos sorry just saw your comment now. Not necessary atm, after we provide a landing page for security, we can revisit.